android/app/google-services.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
This file is sensitive and should be excluded.  Please force so that history removes this file

Reply:
I did it but in my current feature branch.
I will also retro-do-it here.
iOS has its own plist, which i already added in the ignore ;)

Reply:
Thanks for that